scripts/steps/terraform.sh

```diff
@@ -108,6 +108,17 @@ function terraformDestroy() {
     terraform destroy -auto-approve
 }
 
+function getCrossRegionInfraOutputs() {
+    VIDEOS_GLOBAL_TABLE=$(terraform output --raw videos_table_name)
```

Comment:
(Question) without `--raw`, I get double quote (`"`) surrounding the output.  

This seems to happen from Terraform 0.14 https://github.com/hashicorp/terraform/issues/26831  

I don't see you use `--raw` in other `terraform output`.  Which version of Terraform you are using? This might be a potential source of bug.

Reply:
I am using a newer version but this particular usage of mine is a leftover habit of the older versions. Well spotted!!